analyzer/server.go

```diff
@@ -179,14 +233,19 @@ func NewServer(port int, router *mux.Router) (*Server, error) {
 
 	pipeline := mappings.NewFlowMappingPipeline([]mappings.FlowEnhancer{gfe})
 
+	flowtable := flow.NewFlowTable()
+
 	server := &Server{
 		Port:                port,
 		Router:              router,
 		TopoServer:          tserver,
 		GraphServer:         gserver,
 		FlowMappingPipeline: pipeline,
+		FlowTable:           flowtable,
 	}
+	server.RegisterStaticEndpoints()
 	server.RegisterRpcEndpoints()
+	go flowtable.AsyncExpire(server.flowExpire, 10*time.Minute)
```

Comment:
Good catch, the Remove should not be called on the callback, as AsyncExpire delete it just after calling the callback.

The callback is here to do something the flow just before it will be removed.
It's could be used to send it to the database for example.